app/src/main/java/uk/co/ribot/androidboilerplate/ui/base/BaseActivity.java

```diff
@@ -34,26 +33,23 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Create the ActivityComponent and reuses cached ConfigPersistentComponent if this is
         // being called after a configuration change.
-        mActivityId = savedInstanceState != null ?
-                savedInstanceState.getLong(KEY_ACTIVITY_ID) : NEXT_ID.getAndIncrement();
-        ConfigPersistentComponent configPersistentComponent;
-        if (!sComponentsMap.containsKey(mActivityId)) {
+        mActivityId = savedInstanceState != null ? savedInstanceState.getLong(KEY_ACTIVITY_ID) : NEXT_ID.getAndIncrement();
```

Comment:
This line is too long, our guidelines state that the length of a line should not exceed 100 characters. https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md#2215-line-length-limit

Reply:
Okey, check again.